src/routes/trip.route.js

```diff
@@ -1,7 +1,9 @@
 import express from 'express';
 import { tripController } from '../controllers';
 import { validate, errorHandler, isLoggedIn } from '../middlewares';
+import * as auth from '../middlewares/authorize';
 import tripSchema from '../validations/tripValidation';
+import approveRequestSchema from '../validations/approveRequestValidation';
```

Comment:
You have already exported the schema in validations index, you can import all schemas at once.
```suggestion
import {tripSchema,approveRequestSchema } from '../validations/tripValidation';
```